src/client/components/hud/Initiative.tsx

```diff
@@ -103,6 +105,11 @@ export function InitiativeHUD() {
   const characterCollection = useServerState((state) => state.characters);
   const playerCollection = useServerState((state) => state.players);
   const rows = entries(initiativeTracker.entries);
+  const alert = useAlert();
+
+  const myCharacters = useServerState(
+    (state) => state.players.entities[myself.id]!.characterIds
+  );
```

Comment:
I might be missing something, but wasn't our ideomatic way to access player information `useMyself("characterIds")`?

Reply:
Hmm, yeah, you're right. We even already get them from the `useMyProps(...myMapProps)`